### ISSUE TITLE
Using a Construction Wand with SecurityCraft blocks

### ISSUE THREAD

**Issue by WiseHorror** [Theta-Dev/ConstructionWand#29]
If you use a Construction Wand to build with SecurityCraft blocks, you will not be the owner of those blocks, hence you are unable to remove them.

**Comment by Theta-Dev** [comment]
Thanks WiseHorror, I have successfully reproduced the issue.
The problem is that the wand does not call the OnBlockPlacedBy method which updates the owner in SecurityCraft.
I've already written a fix and will probably upload a new version next week.

While I was at it I also tried other building helper mods to see if this bug is present there as well.
Seems like both the Blockzapper from Create and the Building Gadget cant place blocks with tile entities at all.
That's the reason why they aren't calling this method in their code.

![2020-11-22_00 13 01](https://user-images.githubusercontent.com/19636565/99889943-5cc49d00-2c5a-11eb-9e6a-1641588b8e62.png)
